### PR TITLE
Delete `make docker` target

### PR DESCRIPTION
**Pull request by pastuxso**
During a setup on a new machine, I noticed the command `make docker` was not working due to a path issue.

@adambabik I was wondering if it's still needed. is it?.

Trying to solve this issue on Linux:

image

### CODE REVIEW

**Comment by sourishkrout** [comment]
@adambabik can you take this review please?

**Review by adambabik — APPROVED** [review]
Good catch! I don't see `make docker` being used anywhere. GoReleaser uses the image but not the Makefile. It's here for the sake of completeness, I guess. Deleting the target sounds like a good option too.

**Comment by pastuxso** [comment]
Let's remove the "make docker" instead. I Agree with you @adambabik.

File: internal/project/teststub/teststub.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ func Setup(t *testing.T, temp string) Path {
 
 	testDataSrc := originalTestDataPath()
 	require.NoError(t, copy.Copy(testDataSrc, temp))
+	Cleanup(t, temp)
```
Comment (adambabik) [inline internal/project/teststub/teststub.go:18]: @pastuxso did you encounter any problem that led to creating this helper? Overall, when using `T.TempDir()` it is cleared automatically when the test case finishes.

Comment (pastuxso) [reply internal/project/teststub/teststub.go:18]: @adambabik When I run the tests on Linux, it's failing, even individually performed.

This is an example:

image

**Pull request merged**